code/modules/reagents/reagents/drink.dm

```diff
@@ -206,21 +206,61 @@
 	adj_sleepy = -2
 	adj_temp = 20
 	taste_description = "bitterness"
+	purge_rate = 2
+	trait_flags = TACHYCARDIC
+
+
+/datum/reagent/consumable/drink/coffee/overdose_process(mob/living/L, metabolism)
+	L.apply_damage(0.2, TOX)
+	L.jitter(2)
+	if(prob(5) && ishuman(L))
+		var/mob/living/carbon/human/H = L
+		var/datum/internal_organ/heart/E = H.internal_organs_by_name["heart"]
+		if(E)
+			E.take_damage(1, TRUE)
+		L.emote(pick("twitch", "blink_r", "shiver"))
+
+/datum/reagent/consumable/drink/coffee/overdose_crit_process(mob/living/L, metabolism)
+	L.apply_damage(0.5, TOX)
+	L.jitter(5)
+	if(prob(5) && L.stat != UNCONSCIOUS)
+		to_chat(L, "<span class='warning'>You spasm and pass out!</span>")
+		L.Unconscious(10 SECONDS)
+	if(prob(30) && ishuman(L))
+		var/mob/living/carbon/human/H = L
+		var/datum/internal_organ/heart/E = H.internal_organs_by_name["heart"]
+		if(E)
+			E.take_damage(1, TRUE)
+
+//nice one jpr
+/datum/reagent/consumable/drink/coffee/atomiccoffee
+	name = "Atomic Coffee"
+	description = "TThis coffee is a brewed drink prepared from roasted seeds and enriched from use in atomic coffemaker. Consume in moderation"
```

Comment:
If you highlight the row, and click suggestion (ctrl+g) you can suggest the change, it also gives you the attribution for the change.

(example)

```suggestion
	description = "TThis coffee is a brewed drink prepared from roasted seeds and enriched from use in atomic coffemaker. Consume in moderation"
```